src/libnnstreamer-edge/nnstreamer-edge-common.c

```diff
@@ -645,9 +646,16 @@ nns_edge_data_destroy (nns_edge_data_h data_h)
   unsigned int i;
 
   ed = (nns_edge_data_s *) data_h;
+  if (!ed) {
+    nns_edge_loge ("Invalid param, given edge data handle is null.");
```

Comment:
This comment is not directly related with PR.
If the user get this error message, It is still difficult for the user to know where the error occurred.
How about printing file name and line or function name?